src/components/app/AppViewRouter.js

```diff
@@ -54,6 +54,7 @@ class AppViewRouterImpl extends PureComponent<AppViewRouterProps> {
     switch (phase) {
       case 'INITIALIZING':
       case 'PROFILE_LOADED':
+      case 'DATA_RELOAD':
```

Comment:
This could be done on a full switch only, instead of when we've already computed it.